Tchap/Modules/Contacts/ContactsCoordinator.swift

```diff
@@ -16,6 +16,10 @@
 
 import Foundation
 
+protocol ContactsCoordinatorDelegate: class {
+    func contactsCoordinatorShowRoom(_ coordinator: ContactsCoordinatorType, roomID: String)
```

Comment:
Name like this would be better:
`func contactsCoordinator(_ coordinator: ContactsCoordinatorType, didSelectRoomID roomID: String)`
or 
`func contactsCoordinator(_ coordinator: ContactsCoordinatorType, wantsToShowRoomWith roomID: String)`